.travis.yml

```diff
@@ -4,18 +4,25 @@ matrix:
   include:
     # scala 2.10
     - jdk: oraclejdk8
-      scala: 2.10.5
-      env: SCALAZ_STREAM_VERSION=0.7.3a
+      scala: 2.10.6
+      env: SCALAZ_STREAM_VERSION=0.8.6 SCALAZ_VERSION=7.1.11
     - jdk: oraclejdk8
-      scala: 2.10.5
-      env: SCALAZ_STREAM_VERSION=0.8.1a
+      scala: 2.10.6
+      env: SCALAZ_STREAM_VERSION=0.8.6a SCALAZ_VERSION=7.2.8
     # scala 2.11
     - jdk: oraclejdk8
       scala: 2.11.8
-      env: SCALAZ_STREAM_VERSION=0.8.1a
+      env: SCALAZ_STREAM_VERSION=0.8.6 SCALAZ_VERSION=7.1.11
     - jdk: oraclejdk8
       scala: 2.11.8
-      env: SCALAZ_STREAM_VERSION=0.7.3a
```

Comment:
We still need scalaz 0.7 i'm afraid (this is the major reason its still in the build). Can we add this back?